lectocal/lectio.py

```diff
@@ -65,14 +65,14 @@ def _get_lectio_weekformat_with_offset(offset):
 
 
 def _get_id_from_link(link):
-    match = re.search("absid=(\d+)", link)
+    match = re.search("absid|ProeveholdId=(\d+)", link)
```

Comment:
Shouldn't this be 
`match = re.search("(absid|ProeveholdId)=(\d+)", link)`
in order to capture both ProeveholdId and retain compatibility with absid?

Reply:
Thank you for the quick response.
You are right about the README, I didn't think about that.
The regexp actually supports both absid and ProeveholdId as it is (I have verifified it). Maybe the group will make it clearer, but then it should probably be
`match = re.search("(?:absid|ProeveholdId)=(\d+)", link)`
in order to create a non-capturing group.
I'll update the PR later.

Reply:
You are right in that it should of course be a non-capturing group, my bad :)

I have just tested your example in both Python 2 and Python 3 - In both cases, no capture will happen with the input "absid=123123":
```python
import re
link = "absid=123123"
match1 = re.search("absid=(\d+)", link)
match2 = re.search("absid|ProeveholdId=(\d+)", link)
print(match1.group(1))
print(match2.group(1))
```
Output in both Python 2 and 3 is
```
123123
None
```

(My initial comment was based on testing both versions in https://regex101.com/ - A very handy site for constructing regexes)